src/05-objects.js

```diff
@@ -35,7 +40,11 @@ function createObjectWithValue(val) {}
  *  createObjectWithKey("left");
  *  //> { left: false }
  */
-function createObjectWithKey(key) {}
+function createObjectWithKey(key) {
+  let obj = {}
+  obj[key] = "anything"
+  return obj;
```

Comment:
You can return an object literal here and make it one line. Start with `return {}` and see if you can fill in what could go into those curly brackets!